.travis.yml

```diff
@@ -54,7 +54,7 @@ matrix:
         # Sphinx is pinned so that new versions that introduce new warnings won't suddenly cause build failures.
         # (Updating the version is fine as long as no warnings are raised by doing so.)
         # The theme used by the docs is stored seperately, so we need to install that as well.
-        - python -m pip install sphinx~=1.6.1 blurb python-docs-theme
+        - python -m pip install sphinx blurb python-docs-theme
```

Comment:
Would it be possible to fix the spelling of separately on the line above this?

Reply:
In fact, I am going to explain.

with `sphinx.ext.doctest` > 1.6.1, there is a new option for the `.. doctest::`, it's `:skipif:` and I use it in the doctests for `library/turtle.rst` because we need to execute the tests if `tkinter` is not installed on the system.

So, it's the reason I use the last version of sphinx.

Also, @JulienPalard has migrated `doc.python.org` to `Sphinx==1.8.1` 

See https://github.com/python/docsbuild-scripts/pull/59/files

Reply:
lol  ok ;-)

Reply:
sorry @csabella I am going to fix the spelling of separately ;-) sorry